docs/pages/build-reference/variables.mdx

```diff
@@ -175,7 +175,7 @@ If you're using a **.env** file for storing your secrets locally, you can use th
 <Terminal
   cmd={[
     '$ eas secret:push --scope project --env-file ./eas/.env',
-    '✔ Creating secrets on account johndoe...',
+    '✔ Creating secrets on account johndoe',
```

Comment:
```suggestion
    '✔ Creating secrets on account johndoe',
```

This is the actual output of the Expo CLI, we probably should not update that manually, besides when we would like to match this to the current CLI output. 

Alternatively, we can think about refactoring those code block, so they do not include the output.

Reply:
I've reverted this change.